templates/taxi/manufacturer_list.html

```diff
@@ -23,10 +25,23 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
```

Comment:
Consider using a form with the `POST` method for the update action to prevent unintended actions from being triggered by simple URL visits.